src/routes/user.routes.js

```diff
@@ -0,0 +1,19 @@
+
+import Authorization from "../controllers/user.controller";
+import checkAuth from "../middlewares/checkauth";
+import express from "express";
+const router = express.Router();
+
+ router.use((req, res, next) => {
+        res.header("Access-Control-Allow-Headers",
+        "x-access-token, Origin, Content-Type, Accept"
+        );
+        next();
+    });
+ 
+    router.get("/test/all", checkAuth, Authorization.allAcess);
```

Comment:
use `post`  as route url not `test` for more details follow the guideline on README file